components/applied-filters/applied-core-filters.js

```diff
@@ -1,7 +1,7 @@
 import '@brightspace-ui/core/components/button/button-subtle.js';
 import '@brightspace-ui-labs/multi-select/multi-select-list.js';
 import '@brightspace-ui-labs/multi-select/multi-select-list-item.js';
-import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { css, html, LitElement } from 'lit-element';
```

Comment:
I made this change in a couple of my PRs and then realized after it's not really necessary. These ultimately will end up getting switched over to `import { css, html LitElement } from 'lit'` anyway.